core/templates/pages/community-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -130,6 +160,12 @@ export class ContributionOpportunitiesBackendApiService {
         params, resolve, reject);
     });
   }
+
+  fetchFeaturedTranslationLanguages(): Promise<Object> {
+    return new Promise((resolve, reject) => {
+      this._getFeaturedTranslationLanguages(resolve, reject);
+    });
+  }
```

Comment:
Hmm, instead of this, I feel it would be simpler to convert this public method and the private helper method into an [async](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await) function.

Reply:
@sagangwee I think async goes against convention of the code base, I don't see any other API calls using async/await

Reply:
That's probably due to the codebase being old, so most of the API usages are still using Promise chains. If you want to use a Promise chain here though, can't you just return the result of the `then()` call rather than explicitly invoking the success and failure callbacks? Then, you can inline the `fetchFeaturedTranslationLanguages` method instead of using a private helper method.

Reply:
Done